sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/util/BinaryData.java

```diff
@@ -144,39 +159,26 @@ public static BinaryData fromStream(InputStream inputStream) {
             .flatMap(bytes -> Mono.just(fromBytes(bytes)));
     }
 
-    /**
-     * Create {@link BinaryData} instance with given data and character set.
-     *
-     * <p><strong>Create an instance from String</strong></p>
-     * {@codesnippet com.azure.core.experimental.util.BinaryDocument.from#String}
-     *
-     * @param data to use.
-     * @param charSet to use.
-     * @throws NullPointerException if {@code inputStream} is null.
-     * @return {@link BinaryData} representing the binary data.
-     */
-    public static BinaryData fromString(String data, Charset charSet) {
-        Objects.requireNonNull(data, "'data' cannot be null.");
-
-        return new BinaryData(data.getBytes(charSet));
-    }
-
     /**
      * Create {@link BinaryData} instance with given data. The {@link String} is converted into bytes  using
-     * {@link StandardCharsets#UTF_8} character set.
+     * {@link StandardCharsets#UTF_8} character set. If {@code null} data is provided , it will be converted into
```

Comment:
```suggestion
     * the UTF-8 character set. If {@code null} data is provided , it will be converted into
```

`StandardCharsets#UTF-8` is just an implementation of `Charset`, don't need to call out implementation handling.